WTWSMS/localisation/WtWSMS_trade_routes.csv

```diff
@@ -0,0 +1,2 @@
+#CODE;ENGLISH;FRENCH;GERMAN;;SPANISH;;;;;;;;;x;
+amber_route;Amber Road;;;;;;;;;;;;;x;
```

Comment:
Add this to misc, it doesn't need a separate file as the mod only adds one new trade route.

Reply:
ok